### PR TITLE
A few fixes for Clean City dialog

### PR DESCRIPTION
**Pull request by mikeklaas**
This is my first time contributing so please let me know if you'd prefer contributions in another form (separating out by file, or by area, or by types of I tried to only make changes where the intent was 100% obvious, though admittedly it's a bit hard to know at all times.  I've played through the entire game, but do not know Russian

### CODE REVIEW

File: text/english/dialog/vcchead.msg [hunk 1]
```diff
@@ -26,7 +26,7 @@
 {130}{}{[Leave]}
 {131}{}{You call yourself a colonel, but you look and talk like a hillbilly in the bar.}
 
-{160}{}{Bring me that device, before I completely lose my patience.}
+{160}{}{Bring me that device before I completely lose my patience.}
```
Comment (mikeklaas) [inline text/english/dialog/vcchead.msg:29]: I removed many commas which were ungrammatical.  I didn't change any that were purely stylistic/arguable.

File: text/english/dialog/vccopzam.msg [hunk 2]
```diff
@@ -54,29 +54,29 @@
...
 {178}{}{You're telling me that, the "Wasteland disease" is not real? I'm a doctor with many years of experience, we have seen the whole villages die from that. Just like in Tonopah or Mesquite. Have you ever been there?}
-{179}{}{Hmm. I heard something about Tonopah. Something about mad people, consuming their own flesh... Or it was really the whole village, which just died out there... But since you're a doctor, I have to believe you. First, write a receipt that you vouch for their diagnosis.}
+{179}{}{Hmm. I heard something about Tonopah. Something about mad people, consuming their own flesh... Or it was really the whole village, which just died out there... But since you're a doctor, I have to believe you. First, write a receipt that you vouch for the diagnosis.}
 
 #����� ����������� � ������� �������
 {180}{}{Is there any work here for a private detective?}
-{181}{}{Usually we don't hire anyone and deal by ourselves. But there is a few cases, where your help could be appreciated.}
-{182}{}{Interesting. What is it?}
+{181}{}{Usually we don't hire anyone and work by ourselves. But there are two cases where your help would be appreciated.}
```
Comment (mikeklaas) [inline text/english/dialog/vccopzam.msg:75]: "two" is needed for "what's the other one" to make sense

File: text/english/dialog/vccopzam.msg [hunk 3]
```diff
@@ -54,29 +54,29 @@
...
 {172}{}{I will pay. Here's the money.}
 {173}{}{No use talking to you. See you.}
 {174}{}{Prisoner Murphy! You're free to go.}
 {175}{}{Thank you.}
 {176}{}{The money is more important than your health? And if you get infected?}
 {177}{}{Ha-ha-ha, you don't say? This trick is a hundred years old. I've heard everything: "Wasteland disease", "shakes", "Narg syndrome"... What did you come up with this time?}
 {178}{}{You're telling me that, the "Wasteland disease" is not real? I'm a doctor with many years of experience, we have seen the whole villages die from that. Just like in Tonopah or Mesquite. Have you ever been there?}
-{179}{}{Hmm. I heard something about Tonopah. Something about mad people, consuming their own flesh... Or it was really the whole village, which just died out there... But since you're a doctor, I have to believe you. First, write a receipt that you vouch for their diagnosis.}
+{179}{}{Hmm. I heard something about Tonopah. Something about mad people, consuming their own flesh... Or it was really the whole village, which just died out there... But since you're a doctor, I have to believe you. First, write a receipt that you vouch for the diagnosis.}
```
Comment (mikeklaas) [inline text/english/dialog/vccopzam.msg:71]: "receipt" doesn't make much sense in this context ("affidavit" would be better), but "receipt" is used all throughout Clean City quests and should probably be changes holistically.

**Comment by KeyboardGecko** [comment]
okay, cool

**Pull request merged**